keyboards/keychron/q2/ansi_encoder/keymaps/mkillewald/readme.md

```diff
@@ -0,0 +1,73 @@
+    - Activated by double tapping left Shift
+        - can change activation to tapping both left and right Shift by replacing define DOUBLE_TAP_SHIFT_TURNS_ON_CAPS_WORD with #define BOTH_SHIFTS_TURNS_ON_CAPS_WORD in config.h
+    - Left Shift, left Control and left Option (or left Windows key) will light when Caps Word is enabled
+        - #define CAPS_WORD_INDICATOR_COLOR [color] in config.h to set the backlight color used for the indicator when Caps Word is enabled (default: red)
+        - remove #define CAPS_WORD_LIGHT_LOWER_LEFT_CORNER from config.h if you wish for the indicator to only light left Shift
+
+- Caps Lock RGB indicator
+    - the Caps Lock key will light when Caps Lock is enabled with the following options:
+        - #define CAPS_LOCK_INDICATOR_COLOR [color] in config.h to set the backlight color used for the indicator when Caps Lock is enabled (default: red)
+        - Fn2+Z (keycode: LTTOG) will toggle lighting the TAB key when Caps Lock is enabled. This is useful with non backlit keycaps/legends. (default: off)
+        - Fn2+X (keycode: LATOG) will toggle lighting all the alpha keys when Caps Lock is enabled. (default: off)
+
+- Dynamic Fn layer RGB indicator
+    - When either Fn1 or Fn2 is held down, any keys defined on the Fn layers in this firmware or in VIA will be highlighted with the following options:
+        - #define FN1_LAYER_COLOR [color] in config.h to set a static color for defined keys on Fn1 layer (default: orange)
+        - #define FN2_LAYER_COLOR [color] in config.h to set a static color for defined keys on Fn2 layer (default: green)
+        - Fn2+C (keycode: TKTOG) will toggle turning off RGB for keys with no definition (default: RGB off)
+        - Fn2+V (keycode: FCTOG) will toggle lighting the defined Fn layer keys with the static color set with FN1_LAYER_COLOR or FN2_LAYER_COLOR (default: static color off)
+
+All custom keycodes described above can be moved to different keys in VIA by using the following keycodes:
+- CUSTOM(11) = LTTOG (default: Fn2+Z) Caps Lock Light Tab Toggle
+- CUSTOM(12) = LATOG (default: Fn2+X) Caps Lock Light Alpha Toggle
+- CUSTOM(13) = TKTOG (default: Fn2+C) Fn layer Trasnparent Key Toggle
+- CUSTOM(14) = FCTOG (default: Fn2+V) Fn layer Color Toggle
+
+RGB must be toggled on for all indicators to function. If you do not want an RGB mode active but still want the indicators, toggle RGB on and turn the brightness all the way off. The indicators will remain at full brightness.
+
+Please make sure to save any customizations you have made in VIA to a .json file before flashing the firmware. Sometimes it has been necessary to re-apply those changes in VIA after flashing the firmware. If that is the case, you will most likely need to manually add the custom keycodes back in.
+
+## Optional .json Configuration File
+
+[keychron_q2_ansi_knob_configure_mkillewald.json](https://drive.google.com/file/d/1fCnly9thzn6KfIpd2yOrHi0w9CQgaIxn/view?usp=share_link)
```

Comment:
```suggestion
```

Reply:
Can I ask why the link needs to be removed?  Its just doing some useful keymap layout changes that I didn't want to hardcode into the firmware, but if you'd rather I do hardcode those changes I can do that. Just curious as to the reasoning behind this.

Reply:
The .json file has been removed and layout changes previously made by this file are now baked into the firmware.